File: Beecrowd/1185.php
<?php

// lê a operação desejada
$operacao = rtrim(fgets(STDIN));

// lê a matriz M[12][12]
$matriz = array();
for ($i = 0; $i < 12; $i++) {
    for ($j = 0; $j < 12; $j++) {
        $matriz[$i][$j] = floatval(rtrim(fgets(STDIN)));
    }
}

// calcula a soma ou a média dos valores acima da diagonal secundária
$soma = 0;
$contagem = 0;
for ($i = 0; $i <= 10; $i++) {
  for ($j = 0; $j <= 10 - $i; $j++) {
      $soma += $matriz[$i][$j];
      $contagem++;
  }
}

// imprime o resultado com uma casa decimal
if ($operacao == 'S') {
    echo number_format($soma, 1, '.', '') . "\n";
} elseif ($operacao == 'M') {
    echo number_format($soma / $contagem, 1, '.', '') . "\n";
}

?>

File: Beecrowd/1185.c
#include <stdio.h> 

int main () {
    double m[12][12];
    double soma = 0;
    int i, j, contagem;
    char operacao;

    scanf("%s", &operacao);

    for (i = 0; i < 12; i++)
    {
       for (j = 0; j < 12; j++) 
       {
            scanf("%lf", &m[i][j]);
       }
    }

     for (i = 0; i <=10 ; i++)
    {
       for (j = 0; j <= 10 - i; j++) 
       {
            soma += m[i][j];
       }
    }
    
     if (operacao == 'M') {
        double media = soma / contagem;
        printf("%.1lf\n", media);
    } else {
        printf("%.1lf\n", soma);
    }


    return 0;




}